MetaMask/metamask-extension
# Changing URLs to help users self-resolve

## Explanation

Currently, there are several (4) locations in the Extension UI which link the users to a Zendesk Knowledge Base article which is very out of date and not helpful, regarding Hardware Wallets. 

This is obviously not optimal, as our users are not able to self-resolve, and are less able to use hardware wallets, which is something we want to promote in terms of security.

To this end, we've created a one-stop article, the Hardware Wallet Hub, to which we should direct users for "learn more about HWW" issues. I've substituted the old URL for the new one in this PR. In the meantime, we have instituted a redirect in Zendesk to ensure that anyone following the old link is routed to the updated article. But we should still change the URL.

## More Information

I've been discussing this with @AlexJupiter, and this is separate from but related to his PR improving HWW-related UI elements: https://github.com/MetaMask/metamask-extension/pull/14738

### Before

This is the old URL: https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet

### After

This is the new URL: https://metamask.zendesk.com/hc/en-us/articles/4408552261275

## Manual Testing Steps

There were four instances of this URL across the codebase in these three files; they should all link to that ^^ URL ending in 1275. There was one already-existing instance of that link, so now there are a total of 5 instances of it.

ui\pages\create-account\connect-hardware\index.js
ui\pages\create-account\connect-hardware\select-hardware.js
ui\pages\settings\advanced-tab\advanced-tab.component.js:
